Pinta.Core/Actions/HelpActions.cs

```diff
@@ -47,10 +47,10 @@ public HelpActions ()
 			fact.Add ("Menu.Help.Translate.png", new Gtk.IconSet (PintaCore.Resources.GetIcon ("Menu.Help.Translate.png")));
 			fact.AddDefault ();
 
-			Website = new Gtk.Action ("Website", Catalog.GetString ("Pinta Website"), null, "Menu.Help.Website.png");
-			Bugs = new Gtk.Action ("Bugs", Catalog.GetString ("File a Bug"), null, "Menu.Help.Bug.png");
-			Translate = new Gtk.Action ("Translate", Catalog.GetString ("Translate This Application"), null, "Menu.Help.Translate.png");
-			About = new Gtk.Action ("About", Catalog.GetString ("About"), null, Stock.About);
+			Website = new Gtk.Action ("Website", string.Format("{0} {1}", PintaCore.ApplicationName, Catalog.GetString ("_Website")), null, "Menu.Help.Website.png");
```

Comment:
The format string needs to be translatable (`string.Format (Catalog.GetString ("{0} _Website"))`) since grammar rules can be very different in other languages.

You may want to place a comment (prefixed with "Translators:" so that our automated tools pick it up) on the preceding line to explain to translators that the application name will be substituted there, since the meaning of the string may be confusing otherwise.